README.md

```diff
@@ -76,7 +76,7 @@ matrixClient.clientOpts = {
 const ydoc = new Y.Doc();
 const provider = new MatrixProvider(ydoc, matrixClient, {
   type: "alias",
-  alias: "matrix-room-alias",
+  alias: "#matrix-room-alias:matrix.org",
```

Comment:
Oh man I've spent so much time figuring out whether its one or the other in various places using the matrix sdk. Thanks so much for adding this clarification